tools/docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:latest
+RUN apk update
+RUN apk add cargo
+RUN wget https://github.com/mbrubeck/agate/archive/refs/tags/v3.0.2.tar.gz
+RUN tar xzf v3.0.2.tar.gz
+RUN ls /
```

Comment:
Should combine multiple command in a single RUN to reduce the number of layers. And this `ls /` is also not necessary.